Makefile

```diff
@@ -48,27 +56,25 @@ clean-dockerdemo:
 ssh:
 	@vagrant ssh netplugin-node1 || echo 'Please run "make demo"'
 
-unit-test: build
-	CONTIV_HOST_GOPATH=$(GOPATH) CONTIV_HOST_GOBIN=$(HOST_GOBIN) \
```

Comment:
On removing these env vars.

`CONTIV_HOST_GOBIN` and `CONTIV_HOST_GOROOT` are used to mount and use host's go environment inside the vagrant vm. This was done to address unit-test failures (which are built inside the vm, see #31 for some historical context) when their is a version mismatch between host's go environment and the one inside vagrant vm. I think we can get rid of these (and also `HOST_GOBIN` and `HOST_GOROOT`), since now we are always able to depend on the sandbox go environment (either container or vm).

`CONTIV_HOST_GOPATH` is used to mount the host's go path inside the vm. Since we used to build natively this would ensure that go tool dependencies like godep and golint are downloaded just once in the host and reused in the vm. I think we can keep this intact as an alternative to having to pass http_proxy information to the vagrant vm using `CONTIV_ENV`